community/examples/tutorial-starccm-slurm.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: starccm-on-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: starccm-slurm
+  region: us-central1
+  zone: us-central1-c
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  # Example - ./resources/network/vpc
+  - id: network1
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  - id: login-script
+    source: modules/scripts/startup-script
+    kind: terraform
+    settings:
+      runners:
+      - type: shell
```

Comment:
isn't this similar to setting "configure_ssh_host_patterns" to "*", wouldn't it be better set to "star*"? [configure_ssh_host_patterns](https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/scripts/startup-script#input_configure_ssh_host_patterns)

Reply:
Is there a correct way to do this? I am getting prompted to accept keys when I run with `configure_ssh_host_patterns` set to [""]. 
```
 ssh starccmslu-debug-ghpc-1
The authenticity of host 'starccmslu-debug-ghpc-1 (10.0.0.5)' can't be established.
ECDSA key fingerprint is SHA256:ee/7Vve7r85pf6UyzXar41ToSkDviAZ50r5tef8XbBE.
ECDSA key fingerprint is MD5:04:0f:d7:19:67:7c:e6:e7:4c:f3:a1:57:2d:18:2b:e4.
Are you sure you want to continue connecting (yes/no)? 
```
@nick-stroud was the one who suggested the script as written based on a customer who was similarly stuck.

Reply:
That took multiple tries, but I get it now. Seems to work.

Reply:
thanks, I should have been specific and said: configure_ssh_host_patterns: ["star*"], or something like that. Did you update the yaml?

Reply:
The key I was missing is that the script where your comment was is only used by the login node. I needed to create one to use with the compute nodes. A learning event for me!